.github/workflows/milestone-workflow.yml

```diff
@@ -110,6 +110,25 @@ jobs:
             --milestone $MILESTONE_VERSION \
             --assignee curquiza
 
+  create-update-version-issue:
+    needs: get-release-version
+    # Create the changelog issue if the release is not only a patch release
+    if: github.event.action == 'created'
```

Comment:
Why do these lines differ from the upper action?

```
 github.event.action == 'created' && needs.get-release-version.outputs.is-patch == 'false'
 ```
 
 > BTW the comment speaks about the changelog :)

Reply:
Because I want this issue to be created even if we do a patch release, which is not the case for this other issues